mycroft/util/__init__.py

```diff
@@ -109,5 +111,21 @@ def connected(host="8.8.8.8", port=53, timeout=3):
             return False
 
 
+def create_signal(signalName):
```

Comment:
Will that work across processes?

An alternative might be to reimplement the current signaling system using a dict and the messagebus.

A module along the lines of:
```python
from mycroft.messagebus.client.ws import WebSocketClient
from mycroft.messagebus.message import Message
from time import time

_signal_dict = {}

ws = WebSocketClient()

def _set_signal_handler(message):
    sig = message.data['name']
    _signal_dict[sig] = time()

def _clear_signal_handler(message):
    sig = message.data['name']
    _signal_dict[sig] = False


def create_signal(signal_name):
    ws.emit(Message('mycroft.signal.set', data={'name': signal_name}))

def check_for_signal(signal_name, sec_lifetime=0):
    signal = _signal_dict[sig].get(signal_name, False)
    if signal == False:
        return False
    elif sec_lifetime != -1 and signal + sec_lifetime < time.time():
        ws.emit(Message('mycroft.signal.clear', data={'name': signal_name}))
    return True

ws.on('mycroft.signal.set', _set_signal_handler)
ws.on('mycroft.signal.clear', _clear_signal_handler)
```

would be compatible with the current signaling system.